apps/prairielearn/src/pages/authLogout/authLogout.js

```diff
@@ -21,7 +21,7 @@ router.get('/', function (req, res, _next) {
   const redirect = req.query.redirect;
   if (redirect && typeof redirect === 'string') {
     res.redirect(decodeURIComponent(redirect));
-  } else if (res.locals.authn_provider_name === 'Shibboleth') {
+  } else if (res.locals?.authn_provider_name === 'Shibboleth') {
```

Comment:
Oops, I guess that doesn't really fix it because it will never trigger when it's ahead of authn in server.js.

Reply:
Zero impact for us, we're using per-institution SAML auth in production instead of the single hardcoded Shib provider. I think Berkeley might be using this Shib code, let me check if they're ok with this change.